app/lib/esClient.js

```diff
@@ -33,11 +50,28 @@ async function getGp(id) {
   }
 }
 
+function getHitsResult(result, code) {
+  const hits = result.hits && result.hits.hits;
```

Comment:
How come both `result.hits` and `result.hits.hits` need to be checked?

Reply:
The ES structure is result.hits.hits, the check prevents a run time error occurring of hits is undefined.